resources/UserValidation.ts

```diff
@@ -0,0 +1,50 @@
+import { UserActions } from "./UserActions";
+
+export class UserValidation extends UserActions{
```

Comment:
Why do you think UserValidation should extend the UserActions ?

Reply:
because i used a method from UserActions in UserValidation, but i've changed this in the latest commit, now it isn't extending UserActions anymore